src/decode.rs

```diff
@@ -4237,19 +4237,12 @@ pub(crate) unsafe fn rav1d_decode_tile_sbrow(
     // backup t->a/l.tx_lpf_y/uv at tile boundaries to use them to "fix"
     // up the initial value in neighbour tiles when running the loopfilter
     let mut align_h = f.bh + 31 & !31;
-    slice::from_raw_parts_mut(
-        f.lf.tx_lpf_right_edge[0],
-        (align_h * tile_col + t.by + sb_step).try_into().unwrap(),
-    )[(align_h * tile_col + t.by).try_into().unwrap()..][..sb_step.try_into().unwrap()]
+    f.lf.tx_lpf_right_edge_y[(align_h * tile_col + t.by).try_into().unwrap()..]
+        [..sb_step.try_into().unwrap()]
```

Comment:
```suggestion
    f.lf.tx_lpf_right_edge_y[(align_h * tile_col + t.by) as usize..]
        [..sb_step as usize]
```

You can change these to `as usize`s instead of `.try_into().unwrap()`s by the way.  I had done that out of precaution since it was being used as a length in `slice::from_raw_parts_mut` and if it overflowed, it'd be UB, as opposed to an out-of-bounds slicing panic.

Same for the others like this.